flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/utils/TableTestBase.scala

```diff
@@ -1644,6 +1625,34 @@ object TableTestUtil {
     tEnv.registerTable(name, table)
   }
 
+  // -- Connector utilities
+
+  /**
+   * Create a blackhole sink [[TableDescriptor]]
+   */
+  def createBlackHoleSinkDescriptor(
+    fieldNames: Array[String],
```

Comment:
let's not offer this method for now, it is nicer to keep name and type next to each other. btw we can also offer `createBlackHoleSinkDescriptor(DataTypes.FIELD(...)*)`